web-api/src/business/useCases/docketEntry/updateDocketEntryMetaInteractor.test.ts

```diff
@@ -600,6 +598,7 @@ describe('updateDocketEntryMetaInteractor', () => {
   });
 
   it('should update the document pending status and the automatic blocked status of the case when setting pending to true', async () => {
+    getCasesByDocketNumbers.mockResolvedValue([MOCK_CASE]);
```

Comment:
Not a big deal at all, but ideally we'd either mock this in a beforeEach/All, or use a mockResolvedValueOnce. This mocks it for all subsequent tests